util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -687,4 +716,3 @@ macro (seqan_register_tests)
         endif (IS_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/${ENTRY})
     endforeach (ENTRY ${ENTRIES})
 endmacro (seqan_register_tests)
-
```

Comment:
Here were two empty lines, thus there is still an empty line. Github would complain if it doesn't have an empty line at the end of a file.